Source/JavaScriptCore/jit/ExecutableAllocator.cpp

```diff
@@ -86,6 +86,14 @@ namespace JSC {
 
 using namespace WTF;
 
+#if OS(DARWIN) && CPU(ARM64)
+// We already rely on page size being CeilingOnPageSize elsewhere (e.g. MarkedBlock).
+// Just use the constexpr CeilingOnPageSize for better efficiency.
+#define EXECUTABLE_PAGE_SIZE CeilingOnPageSize
+#else
+#define EXECUTABLE_PAGE_SIZE WTF::pageSize()
```

Comment:
I wonder if we should use `executablePageSize()` function instead of macro (since usually we use macro values for constexpr constant, but in this case, one of them are function call).
```cpp
#if OS(DARWIN) && CPU(ARM64)
static inline constexpr size_t executablePageSize() { return CeilingOnPageSize; }
#else
static inline size_t executablePageSize() { return WTF::pageSize(); }
#endif

Reply:
Sounds good.  I will apply this.